lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
The problem here is that there are cases where you want to sort on amounts.  I think this one ought to be looked at more deeply since it represents a difference in the output when you click on a header link.